modules/campaignion_email_protest/campaignion_email_protest.install

```diff
@@ -1,21 +1,13 @@
 <?php
 
 /**
- * @file
- * campaignion_email_protest.install
+ * Enable the newly separated content-type module.
```

Comment:
"newly separated" will probably not be an appropriate description after a while…